e2e/tests/121-Portal/ViewPaProfilePage/ViewActivityOverview.spec.ts

```diff
@@ -0,0 +1,119 @@
+import HomePage from '@121-e2e/pages/Home/HomePage';
+import LoginPage from '@121-e2e/pages/Login/LoginPage';
+import RegistrationDetails from '@121-e2e/pages/RegistrationDetails/RegistrationDetailsPage';
+import TableModule from '@121-e2e/pages/Table/TableModule';
+import {
+  amountVisa,
+  paymentNrVisa,
+  programIdVisa,
+  registrationVisa as registrationVisaDefault,
+} from '@121-service/seed-data/mock/visa-card.data';
+import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { ProgramPhase } from '@121-service/src/shared/enum/program-phase.enum';
+import { StatusEnum } from '@121-service/src/shared/enum/status.enum';
+import { waitFor } from '@121-service/src/utils/waitFor.helper';
+import {
+  changePhase,
+  doPayment,
+  waitForPaymentTransactionsToComplete,
+} from '@121-service/test/helpers/program.helper';
+import {
+  awaitChangePaStatus,
+  importRegistrations,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { test } from '@playwright/test';
+import data from '../../../../../121-platform/interfaces/Portal/src/assets/i18n/en.json';
+import Helpers from '../../../pages/Helpers/Helpers';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Set WhatsApp-number for ALL tests in this suite only
+  const registrationVisa = {
+    ...registrationVisaDefault,
+    whatsappPhoneNumber: registrationVisaDefault.phoneNumber,
+  };
+
+  await resetDB(SeedScript.nlrcMultiple);
+  accessToken = await getAccessToken();
+  await waitFor(2_000);
```

Comment:
Is the underscore a typo? Also, what are we waiting for? Maybe a comment would help

Reply:
This one is taken from API test suite and because of asynchronous methods there are this hardcoded waits. Honestly I don't like them and tried to remove them in one test but it causes some errors.